internal/client/workspace/policy/policy_resource.go

```diff
@@ -6,9 +6,9 @@ SPDX-License-Identifier: MPL-2.0
 package policyworkspaceclient
 
 import (
-	"github.com/vmware-tanzu/terraform-provider-tanzu-mission-control/internal/client/transport"
-	"github.com/vmware-tanzu/terraform-provider-tanzu-mission-control/internal/helper"
-	policyworkspacemodel "github.com/vmware-tanzu/terraform-provider-tanzu-mission-control/internal/models/policy/workspace"
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/client/transport"
```

Comment:
rebase ur feature branch, this is not a new change for this commit

Reply:
This is a new change @shreyassreenivas , as in my feature branch we already added the client and models code and those had the same vmware-tanzu as the import path.